rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,501 @@
+[<span class="underline">here</span>](https://spacetx-starfish.readthedocs.io/en/latest/usage/index.html).
+The exact specification of the image data format will depend upon the
+delivery date that the DCP requests for this prototype.
+
+##### Pipeline Recipes
+
+The parameterization of an imaging pipeline depends on the assay in
+question, the microscope used to capture the data, the density of spots
+in the tissue, and the tissue being analyzed. For example, if the same
+assay is run a second time at twice the magnification, spots would be
+expected to have twice the radius, which would need to be communicated
+to the spot-finding algorithm.
+
+Human analysts are very good at selecting these parameters, but
+processes to fit parameters to all datasets across assays, tissues, and
+microscopes have not yet been developed. To solve this problem, starfish
+exposes the ability for users to pilot starfish on small subsets of
+their data in a local environment. These analyses generate parameter
+sets ("pipeline recipes") which are stored in json and validated by
+*starfish*. These parameters are then used to run *starfish* across the
+complete dataset, in a cloud-based environment.
+
+#### Output Data Formats
+
+Image-based transcriptomics experiments emit three output formats that
+contain identified features, all of which can be emitted in either
+netcdf or zarr file formats, the latter being the format consumed by the
+matrix service. In addition, it has the opportunity to emit image files
+that have been processed to align images, remove autofluorescence and
+enhance spots. All output formats (including SpaceTx image format)
+contain full provenance records of the analysis environment, versions of
+starfish and key dependencies, algorithms that were run, and their
+parameters.
+
+##### [IntensityTable]
+
+Image-based transcriptomics experiments image the same tissue using
+multiple color channels, and after subjecting the tissue to different
+reagents. The physical locations of fluorescent molecules manifest as
+spots in the images, and these spots are detected and aggregated across
+fluorescence channels and time to build codes. In practice, the process
+of detecting and decoding spots contains significant possibilities for
+error, and the IntensityTable supports analysis and QC for this
+process.
+
+##### [SegmentationMask]
+
+In order to assign the spots identified above to individual cells, the
+locations of cells within the images must be identified. Compressed
+segmentation masks are a compact way to represent these data, which
+store, for each cell, the portion of the image that is attributed to
+that cell.
+
+##### [ExpressionMatrix]
+
+*starfish* combines spot calls from the IntensityTable with cell areas
+in the SegmentationMask to produce expression matrices. These are saved
+in zarr format and look identical to the matrices emitted by the Data
+Analysis Service pipelines, with the exception that each cell is
```

Comment:
Change Data Analysis Service pipelines to HCA data processing pipelines (as per charter title)